.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,50 @@
+Contributing to Devguide
+========================
+
+Thank You
+---------
+
+First off, thanks for contributing to the devguide of the Python programming
+language! Even if your contribution is not ultimately accepted, the fact you
+put time and effort into helping out is greatly appreciated.
+
+
+Contribution Guidelines
+-----------------------
+
+Please read this `devguide <https://devguide.python.org/>`_ for
+guidance on how to contribute to this project. The documentation covers
+everything from how to build the code to submitting a pull request. There are
+also suggestions on how you can most effectively help the project.
+
+Please be aware that our workflow does deviate slightly from the typical GitHub
+project. Details on how to properly submit a pull request are covered in
+`Lifecycle of a Pull Request <https://devguide.python.org/pullrequest/>`_.
+We utilize various bots and status checks to help with this, so do follow the
+comments they leave and their "Details" links, respectively. The key points of
+our workflow that are not covered by a bot or status check are:
+
+- All discussions that are not directly related to the code in the pull request
+  should happen on bugs.python.org
+- Upon your first non-trivial pull request (which includes documentation changes),
+  feel free to add yourself to ``Misc/ACKS``
```

Comment:
What or where is ``Misc/ACKS``? Can it be linked to?

Reply:
https://github.com/python/cpython/blob/master/Misc/ACKS